arm9/src/unk_0206015C.c

```diff
@@ -23,10 +24,6 @@ extern THUMB_FUNC u32   MOD06_022451F0(u32);
 
 extern THUMB_FUNC void  PlaySE(u32);
```

Comment:
I am aware that this isn't your code, but I would appreciate it if you could remove the THUMB_FUNCs from these externs as well, don't worry if you can't/don't want to, I can always clean it up myself